src/Core/Currency/CurrencyDataProviderInterface.php

```diff
@@ -47,12 +47,19 @@ interface CurrencyDataProviderInterface
     public function getCurrencies($object = false, $active = true, $group_by = false);
 
     /**
-     * Return raw currencies data from the database reated to the current shop.
+     * Return raw currencies data from the database related to the current shop.
      *
      * @return array Installed currencies
      */
     public function findAll();
 
+    /**
+     * Return raw currencies data from the database regardless of the current shop.
+     *
+     * @return array
+     */
+    public function findAllAvailable();
```

Comment:
This could be misunderstood as "all languages available to install", which we use in other places. I think it would be better just to add an optional parameter to `findAll()` instead.